### PR TITLE
Add "Convenience Command" docs

### PR DESCRIPTION
**Pull request by btmonier**
## Description
This PR:
* Adds a new document: `convenience_commands.md`:
  + Describes supporting commands not found in other pipelines
* Fixes formatting issues with mkdocs:
  + unordered list indentation
  + certain code blocks not rendering

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [x] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Add "Convenience Command" docs
```

### CODE REVIEW

File: docs/convenience_commands.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+``` shell
+phg hvcf2gvcf \
+    --reference-file my/updated/ref/fasta.fa \
+    --hvcf-dir hvcf/directory \
+    --db-path my/phg/db
+    --output-dir output/directory/for/gvcfs
+```
+
+**Parameters**
```
Comment (lynnjo) [inline docs/convenience_commands.md:52]: Add optional parameter "conda-env-prefix".  This currently exists, it just isn't shown in the help message (are optional parameters never shown? what controls that?)

File: docs/convenience_commands.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+``` shell
+phg gvcf2hvcf \
+    --bed my/bed/file.bed \
+    --reference-file my/updated/ref/fasta.fa \
+    --gvcf-dir gvcf/directory \
+    --db-path my/phg/db
+```
+
+**Parameters**
```
Comment (lynnjo) [inline docs/convenience_commands.md:26]: I'm going to add conda-env-prefix as we do access AGC with ProcessBuilder() calls, so this needs to be here.  Note that hvcf files are written to the gvcf-dir location.  Perhaps that should be mentioned.

File: docs/convenience_commands.md (hunk 1 above)
Comment (btmonier) [reply docs/convenience_commands.md:52]: I am confused. Downloading the latest version as of this date (`2.3.11.148 `), I am able to see this parameter for `hvcf2gvcf` using the `--help` parameter. Does this not show up for you? Example console output:

Screenshot 2024-07-26 at 4 07 21 PM

Comment (lynnjo) [reply docs/convenience_commands.md:52]: @btmonier It shows up if you use --help.  It doesn't show up if you just type "./phg hvcf2gvcf".  The latter gives a usage command and only shows the required parameters as missing.  I guess that is correct.  I need to be in the habit of typing "help", but I fall back into what I did for TASSEL, which was just type the command.

**Review by zrm22 — APPROVED** [review]

**Review by tcasstevens — APPROVED** [review]

**Review by lynnjo — APPROVED** [review]

**Review by aberthel — APPROVED** [review]

**Pull request merged**